src/main/java/mate/academy/springboot/swagger/repository/ProductRepository.java

```diff
@@ -0,0 +1,13 @@
+package mate.academy.springboot.swagger.repository;
+
+import java.math.BigDecimal;
+import java.util.List;
+import mate.academy.springboot.swagger.model.Product;
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+@Repository
```

Comment:
`@Repostory` is optional here, it will work without it When you extend `JpaRepository`, the annotation is not required